satpy/tests/test_config.py

```diff
@@ -102,9 +102,12 @@ def test_get_plugin_configs(self, iter_entry_points):
         ep.dist.module_path = os.path.join(os.path.sep + 'bla', 'bla')
         iter_entry_points.return_value = [ep]
 
+        import satpy
         from satpy._config import get_entry_points_config_dirs
-        dirs = get_entry_points_config_dirs('satpy.composites')
-        self.assertListEqual(dirs, [os.path.join(ep.dist.module_path, 'satpy_cpe', 'etc')])
+        # don't let user env vars effect results
```

Comment:
Minor nitpick: "affect", not "effect"

Reply:
Ha, thanks. Now you can expect this level of review on your PRs :smiling_imp: